### PR TITLE
Update Nest documentation for Google Home notification setings

### PR DESCRIPTION
**Pull request by gtalarico**
## Proposed change.

Proposal for adding more details on how google home app settings can affect HA ability to use receive camera events.
See community discussion:
https://community.home-assistant.io/t/google-nest-camera-not-sending-events/631491/5

PR is in progress and created to request feedback from maintainers. 
If we like this, I am hoping we can get folks that own some of these other cameras to fill confirm and fill in settings for other models.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository:  n/a
- This PR fixes or closes issue: fixes # n/a

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by allenporter** [comment]
Thank you for taking the time to make these documentation improvements! Very much appreciated! 

One thing we always try to balance is how much to be helpful vs how much maintenance to do in case things change. I also came across https://support.google.com/googlenest/answer/9230439 and I wonder how much of that support article is applicable here to reference? Maybe we can keep some of the structure here, but reduce some of the detail.

**Comment by gtalarico** [comment]
That's a fair point @allenporter, thanks for the feedback.
I wasn't sure if there was a lot of nuance variations between them different camera models 
but agree maintainability. Let me try something simpler

File: source/_integrations/nest.markdown [hunk 1]
```diff
@@ -322,9 +322,38 @@ This feature is enabled by the following permissions:
...
-- *Other permissions in the Nest or Google Home apps*.
+- *Other permissions and notification settings in the Nest or Google Home apps*.
 </div>
 
+### Google Home App Notification Settings
+
+The Google Home App Notifications settings control not only which notifications are sent to your phone,
+but they also affect what gets published to the Pub/Sub feed.
+
+For example, if you enable “away-only notifications”, HASS will only receive events when your phone is away from home.
```
Comment (allenporter) [inline source/_integrations/nest.markdown]:
```suggestion
For example, if you enable *Away-only notifications*, Home Assistant will only receive events when your phone is away from home.
```

**Comment by gtalarico** [comment]
I added a link the troubleshooting guide and removed the "per device" matrix.

I also added a some notes about the "decoupling" of device feed and Pub/Sub feed, and included a 
a table explaining how these key app settings affect HA integration. 
Do you think this is too much?

I find it very counter intuitive that these settings can cause events to not published to p/s feed, 
especially when google home app settings show the events in its internal feed.

PS: Here is another similar report - I am hoping adding these here could reduce issues and troubleshooting time for users.
https://github.com/home-assistant/core/issues/58482#issuecomment-1175586650

**Comment by allenporter** [comment]
I think its a good middle ground how you have it now. Yes, I agree it is confusing and I think this will definitely help a lot of users.

**Comment by gtalarico** [comment]
Sound good. Let me address the CI checks and will tag you for review

**Comment by gtalarico** [comment]
@allenporter all 🟢 , please let me know if you have any additional feedback or change requests

File: source/_integrations/nest.markdown [hunk 2]
```diff
@@ -322,9 +322,40 @@ This feature is enabled by the following permissions:
...
+<!-- textlint-enable -->
+
+{% details "Google Home App Notification Settings" %}
+
+
+| Google Home App Setting  | Notes                                                                                 |
+| ------------------------ | :-----------------------------------------------------------------------------------: |
+| Notifications: Push      | Required for detection event to be published                                          |
+| Notifications: Away-Only | Events will only be published when owner is detected as away from home                |
+| Seen: Motion             | Only `motion` events will be published                                                |
```
Comment (allenporter) [inline source/_integrations/nest.markdown]: Is it "Only" that those events will be published? Seems like this option isn't exclusive with the others

Comment (gtalarico) [reply source/_integrations/nest.markdown]: Oops that's not correct.
Revsion here https://github.com/home-assistant/home-assistant.io/pull/29576/commits/f7060e1dab7ebadcd07c4cc066c74a4e0356b8a4

**Review by allenporter — DISMISSED** [review]
Looks good, thank you.

**Comment by gtalarico** [comment]
Great🔥  
Are PRs merged in batches by maintainers? 
Do we need to update tags ? Not sure if they are automated, but maybe `in-progress` and `needs-rebase` can be removed?

File: source/_integrations/nest.markdown [hunk 3]
```diff
@@ -322,9 +322,40 @@ This feature is enabled by the following permissions:
...
 </div>
 
+### Google Home App Notification Settings
+
+The Google Home App Notifications settings control not only which notifications are sent to your phone,
+but also what gets published to the Pub/Sub feed.
+
+For example, if you enable *Away-only notifications*, Home Assistant will only receive events when your phone is away from home.
+
+Another thing that may not be intuitive, is that seeing the event your device history does not mean it's was published to the feed.
```
Comment (c0ffeeca7) [inline source/_integrations/nest.markdown]:
```suggestion
Another thing that may not be intuitive, is that seeing the event in your device history does not mean it was published to the feed.
```

File: source/_integrations/nest.markdown [hunk 4]
```diff
@@ -322,9 +322,40 @@ This feature is enabled by the following permissions:
...
+However, if you are getting push notifications the settings are likely working.
```
Comment (c0ffeeca7) [inline source/_integrations/nest.markdown]:
```suggestion
However, if you are getting push notifications, the settings are likely working.
```

File: source/_integrations/nest.markdown [hunk 5]
```diff
@@ -322,9 +322,40 @@ This feature is enabled by the following permissions:
...
+
+Note: The exact settings and affect to feed  may vary by camera model or app version.
```
Comment (c0ffeeca7) [inline source/_integrations/nest.markdown]:
```suggestion
Note: The exact settings and effect they have on the feed may vary by camera model or app version.
```

File: source/_integrations/nest.markdown [hunk 6]
```diff
@@ -322,9 +322,40 @@ This feature is enabled by the following permissions:
...
+
+
+If you are still not getting notifications you can read this [troubleshooting guide from Google]<!-- textlint-disable -->
```
Comment (c0ffeeca7) [inline source/_integrations/nest.markdown]:
```suggestion
If you are still not getting notifications, you can read this [troubleshooting guide from Google]<!-- textlint-disable -->
```

File: source/_integrations/nest.markdown [hunk 7]
```diff
@@ -322,9 +322,40 @@ This feature is enabled by the following permissions:
...
+(https://support.google.com/googlenest/answer/9230439#zippy=%2Cyour-camera-detected-something-but-you-didnt-get-a-camera-alert)
+<!-- textlint-enable -->
+
+{% details "Google Home App Notification Settings" %}
+
+
+| Google Home App Setting  | Notes                                                                                 |
+| ------------------------ | :-----------------------------------------------------------------------------------: |
+| Notifications: Push      | Required for any detection event to be published                                      |
+| Notifications: Away-Only | Events will only be published when user is detected as away from home                 |
```
Comment (c0ffeeca7) [inline source/_integrations/nest.markdown]:
```suggestion
| Notifications: Away-Only | Events will only be published when a user is detected as away from home                 |
```

File: source/_integrations/nest.markdown [hunk 8]
```diff
@@ -575,6 +606,8 @@ logger:
 
 - It is recommended to let Home Assistant create the Pub/Sub subscription for you. However, if you would like more control you can enter a `subscriber_id` in the configuration. See [Subscribe to Events](https://developers.google.com/nest/device-access/subscribe-to-events) for more instructions on how to manually create a subscription and use the full subscription name in the Home Assistant configuration e.g. `projects/gcp-project-name/subscriptions/subscription-id`
 
+- *Not receiving camera motion and person events*: assuming the integration is correctly configured (e.g. oauth and SDM API is setup correctly, you can see camera streams, and permissions are correctly set in [Partner Connections Manager](https://nestservices.google.com/partnerconnections)), but you are still not seeing events, it's possible you need to adjust Google Home App settings. See the [Cameras: Automation: Google Home App Settings](#google-home-app-settings)
```
Comment (c0ffeeca7) [inline source/_integrations/nest.markdown]:
```suggestion
- *Not receiving camera motion and person events*: assuming the integration is correctly configured (for example, the oauth and SDM API are set up correctly, you can see camera streams, and permissions are correctly set in [Partner Connections Manager](https://nestservices.google.com/partnerconnections)): If you are then still not seeing events, it's possible you need to adjust the Google Home App settings. Refer to the [Cameras: Automation: Google Home App Settings](#google-home-app-settings) for details.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @gtalarico :+1:

**Pull request merged**